cmd/evm/internal/t8ntool/transition.go

```diff
@@ -292,6 +292,14 @@ func Transition(ctx *cli.Context) error {
 		prestate.Env.Difficulty = calcDifficulty(chainConfig, env.Number, env.Timestamp,
 			env.ParentTimestamp, env.ParentDifficulty, env.ParentUncleHash)
 	}
+	if chainConfig.IsCancun(big.NewInt(int64(prestate.Env.Number)), prestate.Env.Timestamp) {
```

Comment:
> `big.NewInt(int64(prestate.Env.Number)`

seems like this is used a lot, might be nice to assign above ?

Reply:
Myeah, a couple, but I don't think it's worth it. Might do a bigger refactor later IMO

Reply:
I mean, some uses refer to `big.NewInt(int64(prestate.Env.Number))`, some refer to `int64(prestate.Env.Number)`, and some refer to `prestate.Env.Number`

Reply:
Did the bigger refactor, ptal!